user_guide_src/source/installation/upgrade_4xx.rst

```diff
@@ -254,8 +254,8 @@ Upgrading Libraries
     upgrade_images
     upgrade_localization
     upgrade_migrations
-    upgrade_pagination
     upgrade_responses
+    upgrade_pagination
```

Comment:
Why this change? I assume this was alphabetical.

Reply:
Because I changed the page title.
![Screenshot 2024-04-23 6 26 33](https://github.com/codeigniter4/CodeIgniter4/assets/87955/85791f36-e777-4496-a9c8-8abec7a79602)

Reply:
The output class makes more sense from the perspective of a CI3 user.